File: eAnimalShelter/UI/eanimalshelter_desktop/lib/models/location.dart
class Location {
  int? locationId;
  String? name;

  Location({
    this.locationId,
    this.name,
  });

  factory Location.fromJson(
    Map<String, dynamic> json,
  ) {
    return Location(
      locationId: json["locationId"],
      name: json["name"],
    );
  }
}

File: eAnimalShelter/eAnimalShelter.Services/Database/Location.cs
using System.ComponentModel.DataAnnotations;

namespace eAnimalShelter.Services.Database

{
    public class Location
{
    [Key]
    public int LocationId { get; set; }

    [Required]
    [MaxLength(100)]
    public string Name { get; set; } = string.Empty;
}
}
